### DISCUSSION TITLE
露天讲堂 漫谈新知 2024.1.25

### DISCUSSION THREAD

**Discussion by Lewisrxliu** [RoadSideUniversity/ClubHouse2024#52] [Ideas]
https://www.clubhouse.com/room/B2rtRB10GRIyBvBgO:yMkJdGLDZROEfnWY_9W4GX7SywlWiSb5Rzgfal85heU?utm_medium=ch_instant_invite&utm_campaign=lpYfcJN9MC3YSlNJ-t3K_A-1085357

**Comment by guttasingapore** [comment]
三叔：需要定义新知， 什么是新，什么是知。在谈论的过程中，交流和讨论本身就是非常有意义的。浪漫主义者面对新知：什么是对我有意义的？什么是能够挑起兴奋点的新知？
Lewis:  新知可以是新知识，新资讯，新体系，新的思考和洞见。对认识现在，构建未来深具意义。理解：新是动词，永远是动态，切割时间点，未来每一刻都是新的。

**Reply by guttasingapore** [reply]
Yoyo:  只要心是年轻的，身体健康，就永远年轻。
三叔：如何定义新知是正面的美好的？可能是不好的。如何理解以有涯随无涯，殆已？个人目标：安身立命。
《庄子·养生主》：“吾生也有涯，而知也无涯。以有涯随无涯，殆已！已而为知者，殆而已矣！为善无近名，为恶无近刑。缘督以为经，可以保身，可以全生，可以养亲，可以尽年。”。 人的生命是有限的，而知识是无穷的，用有限的人生追求无限的知识 ，会精疲力竭而无所获。
新知的相对概念是旧识。如何破妄？
Lewis：安身立命，自证践行。宇宙熵增不对称不确定，必须寻求新知。归来永远是少年

**Reply by guttasingapore** [reply]
Too:  好奇心很重要。 知道自己不知道，发现自己的无知 -  很重要，也很难。

**Reply by Lewisrxliu** [reply]
> Too:  好奇心很重要。 知道自己不知道，发现自己的无知 -  很重要，也很难。

Yoyo:知道知道、知道不知道；不知道知道、不知道不知道。

**Reply by guttasingapore** [reply]
感谢Dhi老师： 佛教有庞大深刻完整的认知体系， 当下还不断的在新领域发挥影响力。西方主动探索佛教寻求对话，东方哲学内敛与西方哲学颠覆性突破性有所不同， 佛教抑制人类欲望。印度精神层面与未来释放潜能的相关。

**Reply by Dhidhiddd** [reply]
与其说是“抑制欲望”，不如说是探寻“不受制于欲望”的方法。分布式开放体系☺️😊

**Comment by Lewisrxliu** [comment]
共勉
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/35d6dc03-7e72-437a-bd9e-c5298a37128c)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/134b363c-df05-4ec4-abae-eeb562b7ca28)

**Comment by Lewisrxliu** [comment]
今天我接触到的其中一个新知Python debugging: Print, pdb, VCS, ipdb
需要边学边练，知行合一，学用合一
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/8ebbf0b5-5e07-430f-8b57-94d15ee4ca81)
![image](https://github.com/RoadSideUniversity/ClubHouse2024/assets/128250203/18802345-80e7-4696-ac04-1043a1e50bb5)
